berkeley-function-call-leaderboard/bfcl/model_handler/utils.py

```diff
@@ -828,7 +721,7 @@ def decoded_output_to_execution_list(decoded_output):
     return execution_list
 
 
-def retry_with_backoff(error_type, min_wait=6, max_wait=120):
+def retry_with_backoff(error_type, min_wait=6, max_wait=120, **kwargs):
```

Comment:
Adding stop conditions seemed like a good idea. [The retry library supports this](https://tenacity.readthedocs.io/en/latest/#stopping), and adding `**kwargs` to this definition enables the ability for handlers to use this if they wish.